### PR TITLE
[W5.7][W16-1]Kok Yuan

### PR DESCRIPTION
**Pull request by pixelky**

### CODE REVIEW

**Comment by javenseow** [comment]
Could have included the sort in the user guide. Good work nonetheless.

**Comment by GilgameshTC** [comment]
I think adding empty lines in between each command message displayed really helps with readability. Shame it was removed later on!

**Comment by kengwoon** [comment]
Would be better with more test cases. But still good job.

**Comment by ericyjw** [comment]
Good job! Comments can be added to the method to explain what the method do

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (bqnguyen94) [inline src/seedu/addressbook/parser/Parser.java:14]: These style changes are not really relevant to the main purpose of the PR

File: test/java/seedu/addressbook/parser/ParserTest.java [hunk 2]
```diff
@@ -12,17 +12,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (bqnguyen94) [inline test/java/seedu/addressbook/parser/ParserTest.java:15]: Same here

**Review by bqnguyen94 — CHANGES_REQUESTED** [review]
Good job 👍 just some minor changes needed. Also, for this `sort` command you can add more rigorous tests to check if the internal list is really sorted or not, but this is satisfactory for now.

**Pull request closed without merging**